cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go

```diff
@@ -312,6 +318,22 @@ func (client *autoscalingGceClientV1) FetchZones(region string) ([]string, error
 	return zones, nil
 }
 
+func (client *autoscalingGceClientV1) FetchAvailableCpuPlatforms() (map[string][]string, error) {
```

Comment:
This method is not being used yet?

Reply:
It is not. We use autoscalingGceClientV1 also in GKE cloud provider which is not hosted on github.